charts/opentelemetry-demo/values.yaml

```diff
@@ -57,6 +57,10 @@ components:
     ports:
       - name: redis
         value: 6379
+    schedulingRules:
+      nodeSelector: {}
+      affinity: {}
+      tolerations: []
```

Comment:
What about putting it into default section? I believe the most common use case would be to apply the same rules to all the demo components. We can keep `schedulingRules` per component as well which would take precedence over default.

Reply:
Great idea

Reply:
agreed